docs/docs/casper/overview.md

```diff
@@ -57,6 +59,14 @@ The "Logic" folder hosts a suite of mathematical calculations which are primaril
 
 An "Entrypoint" file consolidates all contract entrypoints within a single trait, streamlining the organization of key functionalities. This modular approach enhances code clarity and accessibility, providing a centralized location for developers to locate and understand the various entrypoints available within the contract.
 
+#### Events
+
+In the "Events" section, we define types of notifications that our contract sends out to external systems. These notifications, or events, help external parties keep track of what's happening in our protocol.
```

Comment:
purpose are events:
1. Provide the ability to collect statistical data.
2. Increase the responsiveness of front-end apps. Web apps subscribe to the creation of an event of a specific type